tests/test-with-simulated-input.el

```diff
@@ -157,18 +157,7 @@
             (read-string "Enter a string: "))
          nil)
         (expect my-non-lexical-var
-                :to-be-truthy)))
-
-    (it "should allow interpolation of variables into KEYS"
-      (let ((my-key-sequence "hello")
-            (my-lisp-form '(insert " world")))
-        (expect
-         (with-simulated-input (list
-                                my-key-sequence
-                                my-lisp-form
-                                "RET")
-           (read-string "Enter a string: "))
-         :to-equal "hello world")))))
```

Comment:
Same reason as below; this could be rewritten as
```emacs-lisp
  (with-simulated-input (my-key-sequence my-lisp-form "RET")
```
if you prefer, though.

Reply:
It looks like the correct rewrite is `(with-simulated-input (my-key-sequence (eval my-lisp-form) "RET" ...)`, which is a bit more awkward